packages/nest/package.json

```diff
@@ -30,10 +30,10 @@
     "peerDependencies": {
         "@mobilejazz/harmony-core": "*",
         "@mobilejazz/harmony-typeorm": "*",
-        "@nestjs/common": "^9.0.0"
+        "@nestjs/common": "9.2.*"
```

Comment:
You've done [similar changes in the reference](https://github.com/mobilejazz/harmony-reference/pull/133/files); in this case though you *shouldn't* do the same. You're over-constraining Nest.

- `9.2.*` only allows patch updates (same as `~9.2.0`). So if our project uses anything that isn't `9.2.x` we'll get a warning/error. These won't be allowed: `9.0.x`, `9.1.x`, `9.3.x`…
  - So… we can't upgrade to `9.3.x` until Harmony changes its constraints.
- In this case the previous one was OK: `^9.0.0`.

Reply:
@doup you're right. Totally. I upgraded all dependencies as if it was a project but it shold allow projects with older versions to use it. I'll update this.